src/explore-education-statistics-frontend/src/components/PageMeta.tsx

```diff
@@ -30,6 +30,12 @@ const PageMeta = ({
         content="jWf4Mg_pzTOgXDWccGcv9stMsdyptYwHeVpODHdesoY"
       />
 
+      {process.env.APP_ENV !== 'Production' && (
+        <>
+          <meta name="robots" content="noindex,nofollow" />
```

Comment:
Would suggest we also put this in admin too under it's `public/index.html` file.

This would be applied to every environment, including prod.

Reply:
Certainly happy to, I just didn't think we cared about the admin site from an SEO perspective. Everything except the login page should be giving a 401 anyway I believe so I'm sure its rating is atrocious already :D